external/3rd/library/soePlatform/VChatAPI/utils2.0/utils/Base/serializeClasses.h

```diff
@@ -389,57 +389,45 @@ namespace soe
 	{
 		if (size < sizeof(int64))
 			return 0;
-#if (defined(WIN32) || defined(linux))
-		uint32 low = *(uint32_t *) &data;
-		uint32 high = *((uint32_t *)&data+1);
-#elif defined(sparc)
-		uint32 low = *((uint32 *)&data+1);
-		uint32 high = *(uint32 *)(&data);
-#endif		
-		stream[BYTE1] = (unsigned char)low&0xff;
-		stream[BYTE2] = (unsigned char)(low>>8)&0xff;
-		stream[BYTE3] = (unsigned char)(low>>16)&0xff;
-		stream[BYTE4] = (unsigned char)(low>>24)&0xff;
-		stream[BYTE5] = (unsigned char)high&0xff;
-		stream[BYTE6] = (unsigned char)(high>>8)&0xff;
-		stream[BYTE7] = (unsigned char)(high>>16)&0xff;
-		stream[BYTE8] = (unsigned char)(high>>24)&0xff;
+		stream[BYTE1] = data&0xff;
+		stream[BYTE2] = (data>>8)&0xff;
+		stream[BYTE3] = (data>>16)&0xff;
+		stream[BYTE4] = (data>>24)&0xff;
+		stream[BYTE5] = (data>>32)&0xff;
+		stream[BYTE6] = (data>>40)&0xff;
+		stream[BYTE7] = (data>>48)&0xff;
+		stream[BYTE8] = (data>>56)&0xff;
 		return sizeof(int64);
```

Comment:
VChatAPI is deprecated, isn't it? can we just delete whatever cancer this is

Reply:
Deprecated?  What's being used in replacement?  Discord?  Would be an effort to remove it properly.

Reply:
Nothing is being used as its replacement we don't have the Vivox libs so there's no support for Voice Chat and even to re-add it with something like Discord we wouldn't use the VChatAPI wrapper afaik it'd just be through ChatServer handling.